CornellDataScience/sketchy
# Edge smoothing

**Description**
Implements basic Edge Smoothing functions (opening, closing, denoising, contour thickening).

**Testing**
1. Navigate to the sketchy/ml/edge_smoothing folder
2.  Run model.py.
3.  The file should return a side-by-side comparison between the input image (a spaceship) and its smoothed-out version.

**Notes**
In addition to displaying the two images, the edge_smoothing function also returns a binary byte file of the finished image.
